src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/repository/IndexStatusRepository.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.prisonersearchindexer.repository
+
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.prisonersearchindexer.model.IndexStatus
+
+@Repository
+interface IndexStatusRepository : ElasticsearchRepository<IndexStatus, String>
```

Comment:
odd that it is still an elasticsearch repo rather than an opensearch repo

Reply:
Surprised at how much smaller the [spring data opensearch](https://github.com/opensearch-project/spring-data-opensearch/tree/main/spring-data-opensearch/src/main/java/org/opensearch/data/client/orhlc) library is. I guess it sits on top of [spring-data-elasticsearch](https://github.com/spring-projects/spring-data-elasticsearch/tree/main/src/main/java/org/springframework/data/elasticsearch)?